### ISSUE TITLE
question about the loss

### ISSUE THREAD

**Issue by shi0605** [stevewongv/DSC-PyTorch#2]
Hi! thanks for your excellent work! I have a question about inconsistent definition of loss function. In the original paper, the weight for shadow pixels is count_neg/(count_pos + count_neg),  however, I found in your utils.py:  beta = count_neg/count_pos.  Is my understanding wrong? Or by doing this, you can achieve better result?

**Comment by stevewongv** [comment]
> Hi! thanks for your excellent work! I have a question about inconsistent definition of loss function. In the original paper, the weight for shadow pixels is count_neg/(count_pos + count_neg),  however, I found in your utils.py:  beta = count_neg/count_pos.  Is my understanding wrong? Or by doing this, you can achieve better result? 

Due to limited time for me, I have not enough time to reproduce the original loss function. So I used the another loss from another paper and produced similar results. You can try to reproduce the original loss, and pull request. Thank you!
